# HG changeset patch
# User Colin Macdonald <cbm@m.fsf.org>
# Date 1405379658 -3600
#      Tue Jul 15 00:14:18 2014 +0100
# Branch stable
# Node ID 1197f04eec09708e5890065ea6dff322d3306d66
# Parent  8de35ac99e809023c0698093b00db2e2f40d4085
doc: minor typo fixes and copy-paste error.

* package.txi: Correct typos.

* toeplitz.m: Correct warning message which should refer to "diagonal"
rather than "anti-diagonal".

diff --git a/doc/interpreter/package.txi b/doc/interpreter/package.txi
--- a/doc/interpreter/package.txi
+++ b/doc/interpreter/package.txi
@@ -266,17 +266,17 @@ the package functions were installed.
 @item package/on_uninstall.m
 This is an optional function that is run prior to the removal of a
 package.  This function is called with a single argument, a struct with
 fields names after the data in the @file{DESCRIPTION}, the paths where
 the package functions are installed, and whether the package is currently
 loaded.
 @end table
 
-Besides the above mentioned files, a package can also contain on or
+Besides the above mentioned files, a package can also contain one or
 more of the following directories:
 
 @table @code
 @item package/inst
 An optional directory containing any files that are directly installed
 by the package.  Typically this will include any @code{m}-files.
 
 @item package/src
@@ -354,17 +354,17 @@ License: GPLv3+
 The package manager currently recognizes the following keywords
 
 @table @code
 @item Name
 Name of the package.
 
 @item Version
 Version of the package.  A package version must be 3 numbers separated
-by a dot.
+by dots.
 
 @item Date
 Date of last update.
 
 @item Author
 Original author of the package.
 
 @item Maintainer
diff --git a/scripts/special-matrix/toeplitz.m b/scripts/special-matrix/toeplitz.m
--- a/scripts/special-matrix/toeplitz.m
+++ b/scripts/special-matrix/toeplitz.m
@@ -69,17 +69,17 @@ function retval = toeplitz (c, r)
 
     r = c;
     nr = length (c);
     nc = nr;
   else
     if (! (isvector (c) && isvector (r))) 
       error ("toeplitz: C and R must be vectors");
     elseif (r(1) != c(1))
-      warning ("toeplitz: column wins anti-diagonal conflict");
+      warning ("toeplitz: column wins diagonal conflict");
     endif
 
     nr = length (c);
     nc = length (r);
   endif
 
   if (nr == 0 || nc == 0)
     ## Empty matrix.
